src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -790,7 +837,7 @@ void func_80A75FA0(Actor* thisx, GlobalContext* globalCtx) {
     }
 }
 
-Gfx* func_80A761B0(GraphicsContext* gfxCtx, u8 primR, u8 primG, u8 primB, u8 envR, u8 envG, u8 envB) {
+Gfx* EnIk_UpdateDisplayList(GraphicsContext* gfxCtx, u8 primR, u8 primG, u8 primB, u8 envR, u8 envG, u8 envB) {
```

Comment:
this type of function is common, for example `Gfx_EnvColor`, `EnDivingGame_EmptyDList` or `EnHy_SetEnvColor`

We have yet to name those consistently afaik but this one can probably be something like `EnIk_(Set?)PrimColorEnvColor`